### ISSUE TITLE
Release for 2.1.14 and 2.1.15

### ISSUE THREAD

**Issue by jameswilson** [coreui/coreui#78]
Is there a reason why coreui does not cut a new Release in github when a new version tag is pushed?  

The last release was from February 22, 2019 for version 2.1.12

Maybe there is an opportunity here to both tag a version, create a github release and publish it to npm in a single command in package.json "scripts" section ?

Some links on the topic:

https://developer.github.com/v3/repos/releases/#create-a-release
https://www.npmjs.com/package/release-it
https://www.npmjs.com/package/generate-release

**Comment by xidedix** [comment]
Hi @jameswilson - thanks for the hints!

The latest tagged 2.1.12 version of @coreui/coreui was released on June 12, 2019.
Later merged PRs relate to dev tooling. There was no need to release a new version.

**Comment by xidedix** [comment]
CoreUI templates and libraries are versioned separately.

**Comment by jameswilson** [comment]
@xidedix , apologies. I've tried to install version 2.1.12 but got some dependency errors:

```
$ npm install @coreui/coreui --save

> core-js@3.2.0 postinstall /Users/jameswilson/App/ProcesoElectronico/www/node_modules/core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> @coreui/coreui@2.1.12 postinstall /Users/jameswilson/App/ProcesoElectronico/www/node_modules/@coreui/coreui
> coreui-plugin-npm-postinstall || exit 0

internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/jameswilson/App/ProcesoElectronico/www/node_modules/@coreui/coreui-plugin-npm-postinstall/index.js:4:15)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN @coreui/coreui@2.1.12 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.
npm WARN @coreui/coreui@2.1.12 requires a peer of perfect-scrollbar@^1.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN @coreui/coreui@2.1.12 requires a peer of popper.js@^1.14.3 but none is installed. You must install peer dependencies yourself.
npm WARN bootstrap@4.3.1 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.
npm WARN bootstrap@4.3.1 requires a peer of popper.js@^1.14.7 but none is installed. You must install peer dependencies yourself.
npm WARN pe-sistema@1.0.0 No description

+ @coreui/coreui@2.1.12
updated 2 packages and audited 5 packages in 3.533s
found 0 vulnerabilities



   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.10.2 → 6.10.3       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.10.3   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯


```

Then, I went exploring and saw there were more recent versions than 2.1.12, and one of the commits was called 'dependency updates' 19f058f, so I assumed that having a newer version might resolve these installation issues.  I hope you can understand my reasoning here. 

If this installation error is unrelated to and not resolved by the dependency updates from the aforementioned commit hash, then please let me know and I'll open up a separate issue. 

Thanks.

**Comment by jameswilson** [comment]
@xidedix 🤦‍♂ it appears the dependency updates and template releases were with a separate repository (https://github.com/coreui/coreui-free-bootstrap-admin-template) , which you gracefully point out above, but i didnt understand what you meant, until I inspected my local copy of the repository and realized it wasnt coreui/coreui.

But be that as it may, I'm still having trouble figuring out how to install coreui/coreui into a project. I guess a separate issue makes sense, but let me know.
